docs/Migrating to version 2.0 guide.md

```diff
@@ -404,6 +424,42 @@ Now you have to use it like:
 <ui5-dialog state="Success"></ui5-dialog>
 ```
 
+- `isOpen` and `close` methods are no longer present. Now `open` property can be used instead.
+
+Previously:
+```ts
+let isOpen = dialog.isOpen();
+dialog.close();
+```
+Now:
+```ts
+let isOpen = dialog.open;
+dialog.open = false;
+```
+
+- `show` method is no longer present. `open` property can be used instead.
+
+Previously:
+```ts
+dialog.show();
+```
+Now:
+```ts
+dialog.open = true;
+```
+
+- parameter `preventInitialFocus` from method `show` is added as a property.
```

Comment:
Parameter `preventInitialFocus` from method `show` is added as a property.

Reply:
Done.